Update to TYPO3 v9 LTS

bmack opened the pull request:
Fixes some installation issues, also updates README and best practices on platform.sh with TYPO3 v9

bmack committed:
[TASK] Raised version to TYPO3 v9

bmack committed:
[BUGFIX] Remove unused config check

bmack committed:
[TASK] Adaptions to new CLI installer

bmack committed:
[BUGFIX] Use FIRST_INSTALL file in proper place

bmack committed:
[BUGFIX] Added more information on maintenance

Crell commented on .platform.app.yaml:
```diff
@@ -5,7 +5,7 @@
 
 # The type of the application to build.
-type: php:7.0
+type: php:7.2
```
Not 7.3? :smile:

bmack replied:
not yet... will do separately!

Crell commented on README.md:
```diff
@@ -1,45 +1,47 @@
+
+Refer to [typo3.org](https://typo3.org/) for information on TYPO3 CMS.
+
+This example is maintained by the TYPO3 Community Interest Group (CIG) Platform.sh, which is headed by
```
The grammar in this sentence is a bit awkward.  Like is there a "for" after "CIG" that's missing?

Crell commented:
Nothing else besides the above jumps out at me as problematic.  Also, congrats on the release. :smile:

ksjogo commented:
Looks reasonable to me, but am not using TYPO3 in an capacity at the moment, so better let someone else test.

bmack committed:
[BUGFIX] Fix grammar in README.md

bmack committed:
[TASK] Update TYPO3 core and dependencies

bmack committed:
[TASK] Update packages

* Update TYPO3 Core
* Use platform reader package
* Set up base config / main site config

bmack committed:
[TASK] Raise everything to TYPO3 v10 with custom installer

bmack committed:
[TASK] Update platform.app.yaml to show updated API

bmack committed:
[TASK] Move to public/ folder

bmack committed:
[TASK] Clean up deploy hook

bmack committed:
[BUGFIX] Ensure folders are moved to the correct location

bmack committed:
[TASK] Move logic into installer setup

bmack committed:
[BUGFIX] Fix error when triggering file + folder structure

bmack committed:
[BUGFIX] Exclude PackageStates.php

bmack committed:
[BUGFIX] Fix deployment script to add ENABLE_INSTALL_TOOL earlier

bmack committed:
[BUGFIX] Create symlinks to legacy path folders

bmack committed:
[BUGFIX] Ensure first run works with symlink

bmack committed:
[BUGFIX] Use a mount for writeable config

bmack committed:
[BUGFIX] Move setup to writable non-commited folder and to build hook

bmack committed:
[BUGFIX] Wire config files on build + deploy

From temp folder to var/ folder to ensure that
this happens in two steps (build + deploy)

bmack committed:
[BUGFIX] Fix env variable for main route environment variable

bmack committed:
[BUGFIX] Add URL without escaping to env variable

bmack committed:
[TASK] Updated README

bmack committed:
[BUGFIX] Ensure to update code as well

bmack committed:
[BUGFIX] Also import data when kickstarting the db

bmack committed:
[BUGFIX] Overcome DataHandler bug when importing data

bmack committed:
[BUGFIX] Fix error message when importing data

bmack committed:
[BUGFIX] Boot up TYPO3 for importing data

bmack committed:
[BUGFIX] Try out to import data before admin user is created

bmack committed:
[BUGFIX] Switch to CLI mode for TYPO3 env

bmack committed:
[BUGFIX] Exclude proper config folder

bmack committed:
[BUGFIX] First create admin user

bmack committed:
[BUGFIX] Authenticate CLI user

bmack committed:
[BUGFIX] Activate extensions before admin user is created
